zebrad/src/components/inbound/tests.rs

```diff
@@ -70,7 +70,7 @@ async fn mempool_requests_for_transactions() {
         .await;
     match response {
         Ok(Response::TransactionIds(response)) => assert_eq!(response, added_transaction_ids),
-        _ => unreachable!(format!(
+        _ => unreachable!("{}", format!(
```

Comment:
I think we can let the unreachable do the formatting
```suggestion
        _ => unreachable!(
```